lib/std/c/hermit.zig

```diff
@@ -1,6 +1,6 @@
 pub const pthread_mutex_t = extern struct {
-    inner: usize = ~usize(0),
+    inner: usize = ~0,
```

Comment:
```zig
test "aoeu" {
    var x: usize = ~0;
}   
```

```
/home/andy/tmp/test.zig:2:20: error: unable to perform binary not operation on type 'comptime_int'
    var x: usize = ~0;
                   ^
```

Maybe this can be made to work, but it does not work in master branch today. A strict translation of the source would be

```suggestion
    inner: usize = ~@as(usize, 0),
```